packages/tokens/src/tokens.ts

```diff
@@ -1,8 +1,27 @@
+import { colors } from './colors';
+
 export const tokens = {
   animations: {
-    spin: {
-      value: 'spin 1s linear infinite',
-    },
+    spin: { value: 'spin 1s linear infinite' },
+  },
+  borders: {
```

Comment:
Some reorg in this file, but mostly catching some token color updates that were broken for borders with the change from `accent` to `ink`.